pubsub-mapped-api/src/main/java/com/google/pubsub/driver/Main.java

```diff
@@ -0,0 +1,50 @@
+package com.google.pubsub.driver;
+
+import java.util.Properties;
+
+import com.google.pubsub.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+/**
+ * Created by gewillovic on 8/4/17.
```

Comment:
True, I forgot to remove it before creating the PR.
I think we will need to provide better example classes, this was meant to be a simple driver.
Removed.